src/log4net/Util/SystemInfo.cs

```diff
@@ -209,13 +172,7 @@ public static string EntryAssemblyLocation
     /// change if the runtime is using fibers.
     /// </para>
     /// </remarks>
-    public static int CurrentThreadId
-    {
-      get
-      {
-        return System.Threading.Thread.CurrentThread.ManagedThreadId;
-      }
-    }
+    public static int CurrentThreadId => System.Threading.Thread.CurrentThread.ManagedThreadId;
```

Comment:
Can you please remove the remarks in the xml comments?
They are irrelevant and/or misleading.